client/components/LearnerViewDirective.js

```diff
@@ -72,15 +72,17 @@ tie.directive('learnerView', [function() {
                       <span ng-if="paragraph.isCodeParagraph()">
                         <code-snippet content="paragraph.getContent()">test</code-snippet>
                       </span>
+                      <span ng-if="paragraph.isSyntaxErrorParagraph()">
+                        <a href class="tie-feedback-syntax-error-link"
+                          ng-click="toggleSyntaxErrorHint(this)">
```

Comment:
I think it needs to be 'this' because it refers to that specific scope that the ng-repeat creates?